modules/microsoft.cdn/azure-frontdoor-cdn-profile/1.0/modules/diagnosticsettings.bicep

```diff
@@ -0,0 +1,62 @@
+@sys.description('Name of CDN Profile. For chaining, use output from parent module')
```

Comment:
remove `@sys` - won't comment on the rest of the files

Reply:
Fixed